src/main.rs

```diff
@@ -1,18 +1,24 @@
 #![windows_subsystem = "windows"]
 
-use clafrica::{run, Config};
-use clafrica_wish::Wish;
+use clafrica::{run, Config as ClafricaConfig};
+use clafrica_wish::{prelude::Config as WishConfig, Wish};
 use std::{env, process};
 
 fn main() {
-    let conf = Config::build(env::args()).unwrap_or_else(|err| {
+    let clafrica_conf = ClafricaConfig::build(env::args()).unwrap_or_else(|err| {
         eprintln!("Problem parsing arguments: {err}");
         process::exit(1);
     });
 
-    let frontend = Wish::build(conf.buffer_size);
+    let wish_conf = WishConfig::from_file("./data/wish.toml").unwrap_or_else(|err| {
```

Comment:
We can wait firstly, the [clafrica](https://github.com/pythonbrad/clafrica) to implement the config file before continue the work.